### DISCUSSION TITLE
Issue를 calendar로 등록하는 github actions 아이디어

### DISCUSSION THREAD

**Discussion by jongfeel** [fkdl0048/Mentoring-Project#17] [Ideas]
사실 저번 미팅 때 얘기하면 좋을 수 있는데
얘기 안 해드린 건 하다가 만 프로젝트이고 흑역사라서 그렇습니다.

그런데
만약 공부하면서 해보실 생각이 있다면
제가 생각하고 구현하려는 방향을 readme.md와 issues에 적어 두었으니
한번 참고해 보시면 좋을 수 있다고 판단했습니다.

https://github.com/jongfeel/WebhookToCalendar

---

마음속으로는 언젠가 다시 하면 좋겠다는 생각은 하고 있습니다.
못해서 문제죠

**Comment by fkdl0048** [comment]
좋은 정보 정말 감사합니다..!  
제가 정말 궁금해 하는 걸 너무 명확하게 알려주셔서 방향이 잘 잡히는 것 같습니다..!  

todolist자체를 action으로 관리해서 매일 22시에 issue를 자동 생성하고 닫히면 readme에 기록 후 파일로 저장해서 관리해볼려고 합니다..!
